gravitee-am-service/src/main/java/io/gravitee/am/service/spring/email/EmailConfiguration.java

```diff
@@ -164,4 +146,32 @@ private List<String> initializeAllowList() {
         return allowList;
 
     }
+
+    private Boolean enabled(){
+        return Boolean.valueOf(configuration.getProperty("email.enabled", "false"));
+    }
+
+    private String host(){
+        return configuration.getProperty("email.host" );
+    }
+
+    private String port(){
+        return configuration.getProperty("email.port");
+    }
+
+    private String username(){
+        return configuration.getProperty("email.username", "#{null}");
+    }
+
+    private String password(){
+        return configuration.getProperty("email.password", "#{null}");
+    }
+
+    private String protocol() {
+        return configuration.getProperty("email.protocol:smtp");
+    }
+
+    private String from(){
+        return configuration.getProperty("email.from");
+    }
```

Comment:
**question:** is it useful to redefine method here to read the property instead of updating the getter already present?

Ex:
```
    public String getHost() {
        return host();
    }
  
   private String host(){
        return configuration.getProperty("email.host" );
    }
```

Could become:
```
    public String getHost() {
        return configuration.getProperty("email.host" );
    }
```

Reply:
You are correct, only properties should be enough